frappe/__init__.py

```diff
@@ -231,8 +231,7 @@ def get_site_config(sites_path=None, site_path=None):
 		if os.path.exists(site_config):
 			config.update(get_file_json(site_config))
 		elif local.site and not local.flags.new_site:
-			print("Site {0} does not exist".format(local.site))
-			sys.exit(1)
+			raise IncorrectSitePath("{0} does not exist".format(local.site))
```

Comment:
I think `DoesNotExistError` would make more sense in this context.

Reply:
If I change to DoesNotExistsError I am getting this error
![image](https://user-images.githubusercontent.com/11792643/89507538-43522f00-d7ea-11ea-8845-ee3281f54c1e.png)

Reply:
`DoesNotExistError` is an exception for database access. `IncorrectSitePath` in this case raises werkzeug's `NotFound` exception, which is meant for routing.